### ISSUE TITLE
Feedback: "Navigate backwards to a checked radio button in interaction mode" (Radio Group Example Using aria-activedescendant, Test 15)

### ISSUE THREAD

**Issue by RFischer-FS** [w3c/aria-at#839]
## Description of Behavior

We disagree that a Screen Reader should announce the "group name" and "group end" information when leaving the radio button group via Tab or Shift+Tab. If a user uses tab or shift+tab, the intention is to leave an element to go to the next. An extra confirmation that the group ended or to hear the group's name again isn't necessary. In this situation, it is more beneficial for users to listen to the information about the element they landed on.

## Test Setup

- Test File: [test-15-navigate-backwards-to-a-checked-radio-button-interaction-jaws.collected.html](https://aria-at.netlify.app/tests/radiogroup-aria-activedescendant/test-15-navigate-backwards-to-a-checked-radio-button-interaction-jaws.collected.html)
- AT: JAWS (version undefined)
- Browser: Chrome (version undefined)

**Comment by jscholes** [comment]
@IsaDC Update from meeting with Vispero on November 17, 2022:

We should remove the assertions relating to the group name and role being conveyed, specifically from all tests where the user is moving out of the group.  Hearing the information at that point would be overly verbose and superfluous, hence why we are removing the assertions rather than demoting them to (or keeping them at) optional level.  Note that:

* We should keep assertions relating to group boundaries where they exist, but only for Reading mode  and VoiceOver tests.
* For Interaction mode, we should remove boundary-related assertions entirely, for cases where the user is navigating out of a group.
* This issue technically relates to the wrong testing task, "Navigate backwards to a checked radio button", in which the user's starting point is outside of the group.  Nevertheless, the actionable next steps in this comment relate to those tests where the starting point is inside of the group, and the commands move the user outside of it.
